src/js/components/LoginPage/Login.js

```diff
@@ -86,12 +87,20 @@ class Login extends Component {
   };
 
   render() {
+    var loc_navBarTitle = "KnowYourNation";
```

Comment:
please stick with either camelCase OR under_score naming conventions. don't mix the two